web/.gitignore

```diff
@@ -22,4 +22,6 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
-.env
\ No newline at end of file
+.env
+
+nft-art-generator
```

Comment:
maybe you should create a new module for the `nft-art-generator`

Reply:
good idea in the future we will need generate this arts